subprojects/docs/src/snippets/testing/test-suite-multi-configure-each-extracted/kotlin/build.gradle.kts

```diff
@@ -59,18 +59,22 @@ testing {
 // end::multi-configure[]
 
 val checkDependencies by tasks.registering {
-    dependsOn(
-        configurations.getByName("testRuntimeClasspath"),
-        configurations.getByName("integrationTestRuntimeClasspath"),
-        configurations.getByName("functionalTestRuntimeClasspath")
-    )
+    val testRuntimeClasspath = configurations.getByName("testRuntimeClasspath")
```

Comment:
I think you can replace this with:
```kotlin
    val testRuntimeClasspath: FileCollection = configurations.getByName("testRuntimeClasspath")
```
And use that in both `dependsOn()` and the `doLast {}` lambda.

Reply:
Looks like I can, thanks for the tip!